### PR TITLE
feat: add init lease function

### PR DESCRIPTION
**Pull request by saudelog**

### CODE REVIEW

File: functions/initLease/package.json
Comment (saudelog) [inline functions/initLease/package.json:23]: should this be a layer?

Comment (saudelog) [reply functions/initLease/package.json:24]: will keep it as part of the bundle for now

**Review by inavac182 — APPROVED** [review]

File: functions/initLease/src/index.ts
Comment (inavac182) [inline functions/initLease/src/index.ts:22]: Actually, now that I think twice about this, I don't think we should be user the user from the request, instead I believe it would be safer to use like the token or what is generated at login, so we get the user that is logged and then check validation against the logged user, what do you think?

Comment (inavac182) [reply functions/initLease/src/index.ts:22]: We don't need to block this PR due to this, but something to look at 👍🏼

File: functions/initLease/src/index.ts
Comment (saudelog) [reply functions/initLease/src/index.ts:79]: I have no strong preference, I personally like to see all related logic in a single file as long as it is small and concise to not be jumping across files to understand the flow, but I could see how it may look a bit cleaner as well, will keep it like this for now, if we see a need latter it can be changed

File: functions/initLease/src/index.ts
Comment (saudelog) [reply functions/initLease/src/index.ts:22]: yeah we need to understand a bit better how the authorization and authentication flow will work before making a final call, but from the quick thoughts I had around this, since the `houseID` is part of the request that can cause authorized requests access data from different users, that should not be the case if somehow we get that information from the authorization token some how

**Pull request merged**
